### DISCUSSION TITLE
Wheeled Loader / Shovel for Silo Work

### DISCUSSION THREAD

**Discussion by coatsy35** [Courseplay/Courseplay_FS22#2314] [General]
Are there plans for Silo Work to be able to use a wheeled loader? I tried it with the Claas Torion and Albutt Silage Fork and it didn't' work, the loader barely moved and it kept getting stuck on the heap. This would be really good if it could as this is what's most commonly done in the UK for moving, lifting and levelling crop in a silage pit, then on most farms here they would roll backwards and forwards in lines with the wheels to compress it (It's very unusual to see rear roller compactors on silage pits in the UK).

Currently I have been using a tractor with levelling blade, however this frequently gets stuck and doesn't move out of the way for the silage wagon or trailer either.

Perhaps this is not possible but thought I'd put it out there.

**Comment by Tensuko** [comment]
Thats not possible and not planed.
But we keep that in mind.
However it would be more complicated and on a rather low priority.

**Reply by coatsy35** [reply]
That's fair enough. Thank you for the reply
